diesel_cli/tests/print_schema/print_schema_allow_tables_in_same_query/diesel.toml

```diff
@@ -0,0 +1,3 @@
+[print_schema]
+file = "src/schema.rs"
+allow_tables_in_same_query = "fk_related_tables"
```

Comment:
test should probably be named around that it's testing specifically `fk_related_tables`, not `allow_tables_to_appear_in_same_query` itself.

Reply:
I think the suggestion here is to use `print_schema_fk_related_tables` as test name, which should fit the 63 character limit.